packages/sdk/test/e2e/browser/teamsUserCredential.browser.spec.ts

```diff
@@ -36,28 +36,53 @@ describe("TeamsUserCredential Tests - Browser", () => {
         });
       });
   });
-  after(() => {
+  afterEach(() => {
     sinon.restore();
   });
 
   it("GetUserInfo should success with SSOToken", async function () {
     const credential: TeamsUserCredential = new TeamsUserCredential();
     const info = await credential.getUserInfo();
     assert.strictEqual(info.preferredUserName, env.SDK_INTEGRATION_TEST_ACCOUNT_NAME);
-    assert.strictEqual(info.displayName, env.SDK_INTEGRATION_TEST_ACCOUNT_NAME.split("@", 1)[0]);
+    assert.strictEqual(info.displayName, "Integration Test");
     assert.strictEqual(info.objectId, TEST_USER_OBJECT_ID);
   });
 
   it("GetToken should success with consent scope", async function () {
+    sinon.stub(PublicClientApplication.prototype, <any>"getAccountByUsername").returns(null);
```

Comment:
what will happen if we do not mock `getAccountByUsername`? since it's e2e test, we may expect the functions work as expected.

Reply:
Removed. Thx!